### DISCUSSION TITLE
Admin tools without social login

### DISCUSSION THREAD

**Discussion by halon176** [umputun/remark42#1158] [Q&A] [answered]
Hi, I tried to find, without any success, a way to login as admin without social login.

I'm probably not understanding something.

under ADMIN_SHARED_ID there are logins referring to social accounts, while the id generated by entering an anonymous user name is logically always different

I also found the ADMIN_PASSWD entry but I don't understand how you need to log in to insert it.

Thanks.

**Comment by umputun** [comment, marked as answer]
you can enable email login and use this id as your ADMIN_SHARED_ID

ADMIN_PASSWD is not something for the end-users but rather for the development process to allow easier REST access/testing.
